gateway/webhook/webhook.go

```diff
@@ -138,7 +138,7 @@ func (webhook *HandleT) RequestHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	contentType := r.Header.Get("Content-Type")
-	if strings.ToLower(contentType) == "application/x-www-form-urlencoded" {
+	if strings.Contains(strings.ToLower(contentType), "application/x-www-form-urlencoded") {
```

Comment:
[non-blocking]
Ideally, if we want to be exact, we should use [mine.ParseMediaType](https://pkg.go.dev/mime#ParseMediaType) to parse `Content-Type` [contents](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types#structure_of_a_mime_type)